docs/DeveloperGuide.adoc

```diff
@@ -780,7 +824,7 @@ public void saveUser(User user, Path filePath) throws IOException {
     XmlFileStorage.saveDataToFile(filePath, new XmlSerializableUser(user));
 }
 ----
-Finally, the `XmlSerializableUser` object is saved to the file using `XmlUtil#saveDataToFile(...)`.
+The converted newly created `XmlSerializableUser` object is saved to the file using `XmlUtil#saveDataToFile(...)` as shown in the code snipplets below:
```

Comment:
Minor grammatical error:
_"The converted newly created"_ should be _"The newly created/converted"_